act_ws_texts.js

```diff
@@ -2186,6 +2186,60 @@ const i18nCfg = {
                         ether_round: "Ether Round",
                         charged_shot: "Charged Shot",
                     },
+                    "PL0100": { // Djeeta
+                        "100": "X",
+                        "110": "(X) X",
+                        "120": "(XX) X",
+                        "121": "(XY) X",
+                        "130": "(XXX) X", "130_desc": "Also (XYX) X",
+                        "131": "(XXY) X",
+                        "200": "Power Raise", "200_desc": "This includes both the uncharged and charged versions of this attack",
+                        "201": "(XYY) Y",
+                        "202": "(XXYY) Y",
+                        "203": "(XYXY) Y",   // Not sure what this is
```

Comment:
To answer ID 201-205 This is just Power Raise based on string it comes from, Power Raise alone is from using it without any combo strings.

201 is from Combo A after doing XYY > Quick Charge Power Raise
202 from Combo B after doing XXYY > Quick Charge Power Raise
203 from Combo C After doing XXXYY or XYXYY > Quick Charge Power Raise
204 from Combo D After doing (XXXXY) Y or (XXYXY) Y > Quick Charge Power Raise
205 from Link After doing Link attack > Quick Charge Power Raise

The ID called to check what condition you did to grant the quick charge effect after meeting one of these specific actions. Theres only been 5 as for